### ISSUE TITLE
TypeError on run

### ISSUE THREAD

**Issue by CCludts** [csurfer/pyheatmagic#2]
Trying this out, I got a TypeError on first try. 
specs:
* anaconda 4.3.1
* python 3.5.2
* windows10 pro (64bit)
* jupyter 4.4.0

![image](https://user-images.githubusercontent.com/18717967/29486521-e65a44b6-84e7-11e7-826f-cb6e41ee5012.png)

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-91c61b03397a> in <module>()
----> 1 get_ipython().run_cell_magic('heat', '', 'def test():\n    50 * 50\n    1 + 2 \n    5 / 4')

C:\Users\User\Anaconda3\envs\poker\lib\site-packages\IPython\core\interactiveshell.py in run_cell_magic(self, magic_name, line, cell)
   2113             magic_arg_s = self.var_expand(line, stack_depth)
   2114             with self.builtin_trap:
-> 2115                 result = fn(magic_arg_s, cell)
   2116             return result
   2117 

<decorator-gen-126> in heat(self, line, cell)

C:\Users\User\Anaconda3\envs\poker\lib\site-packages\IPython\core\magic.py in <lambda>(f, *a, **k)
    186     # but it's overkill for just that one bit of state.
    187     def magic_deco(arg):
--> 188         call = lambda f, *a, **k: f(*a, **k)
    189 
    190         if callable(arg):

C:\Users\User\Anaconda3\envs\poker\lib\site-packages\heat.py in heat(self, line, cell)
     49         tmp_file = 'ipython_cell_input.py'
     50         with open(tmp_file, 'wb') as f:
---> 51             f.write(cell)
     52 
     53         pyheat = PyHeat(tmp_file)

TypeError: a bytes-like object is required, not 'str'
```

**Comment by mikejseay** [comment]
I got the same issue.

- python 3.5.2
- anaconda 4.2.0
- mac OSX sierra
- jupyter 4.2.0

<img width="720" alt="screen shot 2017-08-19 at 8 56 57 am" src="https://user-images.githubusercontent.com/8527007/29486797-6bfe0ec6-84bc-11e7-8b9e-2de0e0878347.png">

**Comment by csurfer** [comment]
@PencilBow @mikejseay This is a known issue which is already fixed in v0.0.2 So updating the package version should resolve this for you. That reminds me that I should add a HISTORY file.
